tracer.go

```diff
@@ -354,3 +357,16 @@ func makeParamsAttribute(args []any) attribute.KeyValue {
 	}
 	return QueryParametersKey.StringSlice(ss)
 }
+
+func findOwnImportedVersion() string {
+	buildInfo, ok := debug.ReadBuildInfo()
+	if ok {
+		for _, dep := range buildInfo.Deps {
+			if dep.Path == tracerName {
```

Comment:
Is this always the full and only path? Or should we use `strings.Contains`?

Reply:
it's always the [module's import path](https://pkg.go.dev/runtime/debug#Module), so it's guaranteed to only change if someone forks the project.

Should also work with [`replaces` directive, because](https://go.dev/ref/mod#go-mod-file-replace):
> if the replacement module has a go.mod file, its module directive **must** match the module path it replaces

But I haven't tested that :see_no_evil:

Reply:
Ok let's give it a try